contracts/WildcardSteward_v3.sol

```diff
@@ -633,24 +620,54 @@ contract WildcardSteward_v3 is Initializable {
         uint256 patronageDueBenefactor = patronageDueBenefactor(benefactor);
 
         if (patronageDueBenefactor > 0) {
-            if (benefactorCredit[benefactor] > 0) {
-                if (patronageDueBenefactor < benefactorCredit[benefactor]) {
-                    benefactorCredit[benefactor] = benefactorCredit[benefactor]
-                        .sub(patronageDueBenefactor);
-                } else {
-                    benefactorFunds[benefactor] = patronageDueBenefactor.sub(
-                        benefactorCredit[benefactor]
-                    );
-                    benefactorCredit[benefactor] = 0;
-                }
-            } else {
-                benefactorFunds[benefactor] = benefactorFunds[benefactor].add(
+            _increaseBenefactorBalance(benefactor, patronageDueBenefactor);
+        }
+
+        timeLastCollectedBenefactor[benefactor] = now;
+    }
+
+    function _increaseBenefactorBalance(
```

Comment:
I'm being a stickler, but I prefer defining functions before they are used not after. (some languages even error if you do this...).

Anyway, not important enough to bother with this.